etc/vector/vector.d/90-output.toml.tmpl

```diff
@@ -110,6 +110,14 @@ source = '''
     encoding.codec = "raw_message"
     encoding.timestamp_format = "rfc3339"
   {{end}}
+{{else if getenv "LOGS_EXPORT_CLOUDWATCH"}}
```

Comment:
I think we should make this a little more structured:

* [Add a new](https://github.com/deskpro/docker-product-base/blob/main/Dockerfile#L166) `LOGS_EXPORT_TARGET` var that can be `stdout|dir|cloudwatch`
* [Around here](https://github.com/deskpro/docker-product-base/blob/main/usr/local/sbin/entrypoint.sh#L91), we can check if `LOGS_EXPORT_TARGET` is empty/unset
  * If it's unset && LOGS_EXPORT_DIR is set, then we can default to `dir`
  * If it's unset && there _isn't_ an export dir, then we default to `stdout`
  * So ^ means the current behaviour is maintained
* In this `90-output.toml.tmpl` file we change the conditions to test on `LOGS_EXPORT_TARGET` more explicitly

Just means in the future we can add more types of output without introducing all new vars.

E.g. I could imagine someone wanting to do `LOGS_EXPORT_TARGET=custom` (meaning none of our default logging would apply) and then mounting their own config file with all their own config.

Reply:
FYI I'm going to fold these changes into a PR I'm building now to do with OPC compatibility